scripts/abscal_run.py

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python
+
+from hera_cal import abscal
+import sys
+import os
+
+a = abscal.abscal_arg_parser()
+args = a.parse_args()
+history = ' '.join(sys.argv)
```

Comment:
history doesn't seem to be used in the function call.

Reply:
not part of this pull-request

Reply:
sorry for the confusion, i just didn't want to lose a bunch of work I had already done when I was asked to make this PR. this code is commented out

Reply:
ah sorry, i meant to comment this out, similar to commenting out the `abscal_run()` function in `abscal_funcs.py`. this is something i am going to put into my next PR, and meant to comment out.